src/config.ts

```diff
@@ -128,23 +130,93 @@ export class CacheConfig {
     self.workspaces = workspaces;
 
     let keyFiles = await globFiles("rust-toolchain\nrust-toolchain.toml");
+
+    hasher = crypto.createHash("sha1");
+
     for (const workspace of workspaces) {
       const root = workspace.root;
       keyFiles.push(
         ...(await globFiles(
-          `${root}/**/Cargo.toml\n${root}/**/Cargo.lock\n${root}/**/rust-toolchain\n${root}/**/rust-toolchain.toml`,
+          `${root}/**/rust-toolchain\n${root}/**/rust-toolchain.toml`,
```

Comment:
P.S. I just realized that `rust-cache` does not include `.cargo/config.toml` in hashing, I will submit a PR to fix this after this PR is merged.

Reply:
[`.cargo/config.toml`](https://doc.rust-lang.org/cargo/reference/config.html#configuration-format) contains some settings that can affect the build, like `patch`, `profile`, `build.rustc-wrapper`, `env`, `registeries`, `target.<triple>` and etc.

Reply:
ah, that is a very good idea indeed!

Reply:
I've opened #149 for this.